oximeter/impl/src/schema/mod.rs

```diff
@@ -263,6 +330,45 @@ impl PartialEq for TimeseriesSchema {
 const TIMESERIES_NAME_REGEX: &str =
     "^(([a-z]+[a-z0-9]*)(_([a-z0-9]+))*):(([a-z]+[a-z0-9]*)(_([a-z0-9]+))*)$";
 
+/// Authorization scope for a timeseries.
+///
+/// This describes the level at which a user must be authorized to read data
+/// from a timeseries. For example, fleet-scoping means the data is only visible
+/// to an operator or fleet reader. Project-scoped, on the other hand, indicates
+/// that a user will see data limited to the projects on which they have read
+/// permissions.
+#[derive(
+    Clone,
+    Copy,
+    Debug,
+    Deserialize,
+    Eq,
+    Hash,
+    JsonSchema,
+    Ord,
+    PartialEq,
+    PartialOrd,
+    Serialize,
+)]
+#[serde(rename_all = "snake_case")]
+pub enum AuthzScope {
+    /// Timeseries data is limited to fleet readers.
+    Fleet,
+    /// Timeseries data is limited to the authorized silo for a user.
+    Silo,
+    /// Timeseries data is limited to the authorized projects for a user.
+    Project,
+    /// The timeseries is viewable to all without limitation.
+    None,
+}
+
+impl AuthzScope {
+    /// Construct a fleet authz scope.
+    pub const fn fleet() -> Self {
```

Comment:
Similar nit as above - not sure this method is worth it

Reply:
Same comment -- I used this for `#[serde(default = "...")]`. Since that's going away, I'll remove this field too.

Reply:
Removed in 5610bb467